sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java

```diff
@@ -409,18 +408,19 @@ private void injectReadinessProbe(ArrayNode arrayNode, Map<String, String> env,
         boolean enableSpring = getNotEmptyValue(env, ENABLE_SPRING_KEY, true, Boolean::parseBoolean);
         boolean enableHealthCheck = getNotEmptyValue(env, ENABLE_HEALTH_CHECK_KEY, false, Boolean::parseBoolean);
 
-        // 原来已经有readinessProbe节点/periodSeconds<=0/未开启健康检查/未开启spring优雅上下线，则不注入
+        // If there is already a readiness Probe node /period Seconds<=0/ If health check is not enabled/spring
```

Comment:
If there is already a readinessProbe node /periodSeconds<=0

Reply:
done